www/shakedetect.js

```diff
@@ -0,0 +1,55 @@
+/*global cordova, module*/
+
+var exec = require("cordova/exec")
+
+/*
+ * Format of the returned value:
+ * {
+ *    "isDutyCycling": true/false,
+ *    "accuracy": "high/balanced/hundredmeters/..",
+ *    "geofenceRadius": 1234,
+ *    "accuracyThreshold": 1234,
+ *    "filter": "time/distance",
+ *    "filterValue": 1234,
+ *    "tripEndStationaryMins": 1234
+ * }
+ */
+
+var TransitionNotification = {
+    INCIDENT: 'incident_detected',
```

Comment:
these should match the events that can actually be generated from the transition code
doesn't make sense to have `NO_INCIDENT` because that is the default state, not a transition.